.claude/skills/court-vision-design/colors_and_type.css

```
@@ -0,0 +1,298 @@
+
+  --content-max: 1600px;   /* SectionContainer.tsx max-width */
+
+  /* ====================================================================
+     TYPE — Families
+     ==================================================================== */
+
+  --font-sans:        'Geist', system-ui, -apple-system, 'Segoe UI', sans-serif;
+  --font-mono:        'Geist Mono', ui-monospace, 'SFMono-Regular',
+                      'Menlo', 'Monaco', monospace;
+  --font-handwriting: 'Patrick Hand', 'Caveat', cursive;
+  --font-serif:       Georgia, 'Times New Roman', serif;   /* "View Resume (PDF)" */
```

Stylelint (17.9.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)